### DISCUSSION TITLE
optimum command line settings for SDRPLay

### DISCUSSION THREAD

**Discussion by OE1MWW** [jvde-github/AIS-catcher#158] [General]
Hi all,
did someone find out the optimum command line parameter using a SDRPlay ?

I am comparing RTL-SDR V3 and a SDRPlay. Using the AISCatcher I can decode
only one BaseStation [1] in my vicinity. BaseStation [1] has a RSSI of -73db
BaseStation [2] is not decoded.

Using AIS decoders for RTL-SDR and SDRPlay (downloaded from rfbits.com) I am 
receiving both BaseStations [1] and [2] and seeing the following results:

RTL-SDR: [1] - 87dbm (~ 10uV), [2] -116 dbm (~ 0,3uV)
SDRPLay: [2] - 85dbm (~ 10uV), [2] -113 dbm (~ 0,3uV)
both decodes varies by +/- 2 dbm

As AISCatcher is my preferred tool, wonder what settings I have to change to
make the AISCatcher+SDRPlay more sensitive?

**Comment by jvde-github** [comment]
Hi,

I read some feedbacks that the RTL-SDR somehow works better with the program than the SDRPlay 1A. Not sure if it is Catcher or the Hardware.

On the best settings, it really depends on your system and requires some experimentation.  Perhaps start with LNAstate zero and experiment with GRDB. The default worked well for me on the DX but this included a filter (and hence stronger gain). See also this discussion: https://github.com/jvde-github/AIS-catcher/discussions/126

Not very helpful I am afraid. I am open to suggestions on how to improve....

**Comment by Taxom** [comment]
In addition to the absolute levels of signal, the levels of interference are also affected (the ratio of the signal noise). Therefore, it is important not only gain but also the ability to filter neighboring signals, especially if they can overload the input cascade of the receiver. Do you use the same bitrate on both receivers? Do you use the limitation of the bandwidth(-a xxx - set tuner bandwidth in Hz)? Do you use any filters in front of the receiver?

**Comment by OE1MWW** [comment]
Taxom, thanks for the hint!
- the -a xxx has not been used. What is the best setting for AIS, for example  '-a 25000' (?)
- no filters used and required, have no other strong signals in that frequency range

**Comment by Taxom** [comment]
1) If you use 2 channels A/B should set at least 75kHz (from 161.975-12.5 to 162.025 +12.5)   (-a 75k) but this is without taking into account the drift of the frequency of transmitters and your receiver. I use -a 100 to capture transmitters with a frequency drift.

2) About the absence of transmitters in this frequency range. With the set sample rate 1536k, you capture the frequency range from 161.232 to 162.768 MHz. I use SAW filter TB0436A (https://www.taisaw.com/upload/product/TB0436A%20_Rev.1.0_.pdf) because I have powerful Weatheradio broadcasts in this range (162.400, 162.425, 162.450, 162.475, 162.500, 162.525, and 162.550 MHz.).

3) But this is not all, a powerful signal that does not even fall into the range to which the dongle is configured, is able to overload it. Sources of such powerful signals are FM broadcasting radio stations.

![1536](https://github.com/jvde-github/AIS-catcher/assets/20313019/538c5caa-1300-402c-8039-0d011aadeaab)

**Comment by Taxom** [comment]
This is what a typical distance of the reception of my receiver looks like. Color circles of 10 nautical miles each. At the same time, I have no direct visibility to the north and east.
![Screenshot 2023-09-06 at 04-28-07 AIS-catcher MaxPi_AIS_Toronto](https://github.com/jvde-github/AIS-catcher/assets/20313019/f41bf71b-71fd-45f7-9fcb-f2009babc15c)

**Comment by OE1MWW** [comment]
Taxom, thanks for the info.
- the -a parameter has no impact on the sensivity. Using other software I can decode two base station, AIS-catcher shows just one.
  see my first post
- interresting link for the SAW filter
- what software did you use for the spectrum display ? For such things I am always curious ;-)

**Reply by Taxom** [reply]
-I'm not sure if the "-a" parameter works with SDRPLAY.
-It's just a screenshot from SDRSharp
SAW filter
![photo_2023-09-07_15-54-17](https://github.com/jvde-github/AIS-catcher/assets/20313019/10622a92-158c-4c99-a751-ea365e3ac73c)

**Reply by jvde-github** [reply]
For SDRPLAY, -a is ignored and default of 200K is used

**Comment by jvde-github** [comment]
What settings are you using in this ortac program and with AIS-Catcher/RTL-SDR are you receiving the two base stations?

I tried the MSI SDR with both programs. With `-gs LNASTATE 0 GRDB 32 -s 2000K` I got decent results without a filter. The default settings in Catcher have a bit higher gain so definitely worthwhile to experiment.

**Comment by OE1MWW** [comment]
Jasper, thanks for the hint !
yes, with "-gs LNASTATE 0 GRDB 32 -s 2000K" I am receiving with the MSI.SDR same amount of signals, including both base stations,
like with ortac in default settings. My command line now:
`AIS-catcher -N LAT 48.268991 LON 16.384977 SHARE_LOC on -gs LNASTATE 0 GRDB 32 -s 2000K -o 0 -u 127.0.0.1 29421 -N 8100 PLUGIN_DIR plugins `
